src/test/java/seedu/address/logic/commands/DeleteEventCommandTest.java

```diff
@@ -27,7 +27,7 @@
 
     @Test
     public void execute_validIndexUnfilteredList_success() {
```

Comment:
should we change the test name to `execute_validIndexFilteredList_success()` instead? since the list we are using is the filtered event list here! same for the other relevant tests in this class!